### ISSUE TITLE
Lingering phase_functions output issue

### ISSUE THREAD

**Issue by Siltala** [maxmahlke/rocks#5]
Hi,

I'll add future rocks-related reports here rather than email!
With the latest version, ` rocks phase_function Ceres` now works and produces the expected output (thanks!), but `rocks phase_functions Ceres` (note the plural) which I would expect to print out all the available phase functions, still has broken output for all asteroids:
```
$ rocks phase_functions Ceres
 (1) 
Ceres
┌───┐
│   │
├───┤
│ 1 │
└───┘
```

**Comment by maxmahlke** [comment]
Confirmed, I had not defined the list of columns to print for phase functions. Fixed in version 1.5.17, thanks for reporting!

**Comment by maxmahlke** [comment]
And before you report it: `phase_function`  is currently outputting the verbose output by default, this is fixed in 1.5.18 (not pushed yet).
